nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -742,6 +743,39 @@ in
         '';
       };
 
+    virtualisation.directBoot = {
+      enable = mkOption {
+        type = types.bool;
+        default = !cfg.useBootLoader;
+        defaultText = "!cfg.useBootLoader";
+        description =
+          lib.mdDoc ''
+            If enabled, the virtual machine will direct boot the system
+            by passing the kernel, initrd and relevant parameters to QEMU.
+
+            If you want to use a bootloader, use `virtualisation.useBootLoader`,
+            if you want to test netboot, you might be interested into disabling this.
+
+            This is enabled by default if you don't enable bootloaders.
+            Read more about this feature: <https://qemu-project.gitlab.io/qemu/system/linuxboot.html>.
+          '';
+      };
+      initrd =
+        mkOption {
```

Comment:
It's confusing that this is indented differently to the other option.  Either style is fine, but let's be consistent!

Reply:
> Looks good. I dont particularly love the "use" prefix but maybe there value in keeping it consistent with the current format, i.e. `opt.directBoot` -> `opt.useDirectBoot`.

Hmm, I think people would disagree on this PR and explicitly ask me to regroup the options so I am sparing myself the bikeshedding. :-)

Reply:
Wrong thread

Reply:
This is fixed I believe.